tensorflow/python/kernel_tests/depthwise_conv_op_test.py

```diff
@@ -369,6 +373,7 @@ def testDepthwiseConv2DInputGrad(self):
       print("Testing DepthwiseConv2DInputGrad,", index, "th config:",
             input_size, "*", filter_size, "stride:", stride, "padding:",
             padding)
+      # float16 test is disabled
```

Comment:
Please punctuate comments `The test for float16 is disabled.` I would be in favor of removing that comment however.

Is it really disabled?

Reply:
@drpngx The comments has been updated.